doc/source/tutorials/emulator.rst

```diff
@@ -5,7 +5,7 @@ In this section we are going to explain how to setup a platform for pulse emulat
 and how to use it in Qibolab. To setup a platform we first recommend to have a look at
 :ref:`tutorial_platform`.
 
-We are now going to explain a few differences which are necessary to build an emulation
+We are now going to explain a few distinctive features/elements of an emulation
```

Comment:
Sorry, when I wrote *features/elements* I intended them as synonyms, and left the choice to you.

Though, to be fair, it is even fine like this.